consolidate scattered ex-distribution infra and tools docs (#1267)
sourcegraph/handbook: content/product-engineering/engineering/enablement/delivery/customers/replicas.md

# Customer environment replicas

We maintain separate AWS accounts with Sourcegraph instances and other infrastructure that mimic various customers' environments for testing purposes. See "[Customer environment replicas and managed instances](https://my.1password.com/vaults/dnrhbauihkhjs5ag6vszsme45a/003/ctqvj7zcmdiujmfh2mxzffdlym)" on 1Password for the list.

## Accessing sourcegraph internal infrastrcture

To access this AWS account:

> 1. Request an AWS user ID by asking a member of the #distributioneers team in slack.

Reviewer: ```suggestion
1. Request an AWS user ID by asking in the #it-tech-ops channel in slack.
```

AWS access should be gated by Okta now so I think requests fall on IT.
